client.php

```diff
@@ -14,14 +14,16 @@
 	decimal_column DECIMAL(10,2) NOT NULL DEFAULT 0,
 	float_column FLOAT(10,2) NOT NULL DEFAULT 0,
 	enum_column ENUM('a', 'b', 'c') NOT NULL DEFAULT 'a',
-	date_column DATE NOT NULL DEFAULT CURRENT_DATE,
```

Comment:
we don't support default current_date?

Reply:
@adamziel Actually, [MySQL doesn't](https://onecompiler.com/mysql/43xunf6yf). Only in version 8 with expressions (like `DEFAULT (CURRENT_DATE())`, but we don't support those in `DEFAULT` yet.